frontend/packages/local-storage-operator-plugin/src/components/disks-list/ocs-disks-list.tsx

```diff
@@ -0,0 +1,270 @@
+  cx('pf-m-hidden', 'pf-m-visible-on-2xl'),
+  cx('pf-m-hidden', 'pf-m-visible-on-lg'),
+  cx('pf-m-hidden', 'pf-m-visible-on-xl'),
+  Kebab.columnClass,
+];
+
+const diskHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'path',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Disk State',
+    sortField: 'status.state',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'OCS Status',
+    sortField: '',
+    transforms: [],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'Type',
+    sortField: 'type',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[2] },
+  },
+  {
+    title: 'Model',
+    sortField: 'model',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[3] },
+  },
+  {
+    title: 'Capacity',
+    sortField: 'size',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[4] },
+  },
+  {
+    title: 'Filesystem',
+    sortField: 'fstype',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[5] },
+  },
+  {
+    title: '',
+    sortField: '',
+    transforms: [],
+    props: { className: tableColumnClasses[6] },
+  },
+];
+
+const diskRow: RowFunction<DiskMetadata, OCSMetadata> = ({
+  obj,
+  index,
+  key,
+  style,
+  customData,
+}) => {
+  const { ocsState, dispatch } = customData;
+  const diskName = obj.path;
+  return (
+    <TableRow id={obj.deviceID} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>{obj.path}</TableData>
+      <TableData className={tableColumnClasses[1]}>{obj.status.state}</TableData>
+      <OCSStatus ocsState={ocsState} diskName={diskName} className={tableColumnClasses[1]} />
+      <TableData className={tableColumnClasses[2]}>{obj.type || '-'}</TableData>
+      <TableData className={cx(tableColumnClasses[3], 'co-break-word')}>
+        {obj.model || '-'}
+      </TableData>
+      <TableData className={tableColumnClasses[4]}>
+        {humanizeBinaryBytes(obj.size).string || '-'}
+      </TableData>
+      <TableData className={tableColumnClasses[5]}>{obj.fstype || '-'}</TableData>
+      <OCSKebabOptions
+        diskName={diskName}
+        diskOsdMap={ocsState.alertsMap}
+        isRebalancing={ocsState.isRebalancing}
+        dispatch={dispatch}
+      />{' '}
+    </TableRow>
+  );
+};
+
+export const OCSDisksList: React.FC<TableProps> = React.memo((props) => {
+  const [ocsState, dispatch] = React.useReducer(reducer, initialState);
+
+  const [cephDiskData, cephDiskLoadError, cephDiskLoading] = usePrometheusPoll({
+    endpoint: PrometheusEndpoint.QUERY,
+    query: osdDiskInfoMetric({ nodeName: props.customData.node }),
+  });
+  const [progressData, progressLoadError, progressLoading] = usePrometheusPoll({
+    endpoint: PrometheusEndpoint.QUERY,
+    query: DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS],
+  });
```

Comment:
`usePrometheusQuery` is for dashboards utilization.

Reply:
ah okay. In the future, we should definitely move towards having a dedicated redux sub store for metrics rather than one dedicated for the dashboard. For now, you can resolve this.